cuda/test/base/cuda_executor.cu

```diff
@@ -95,11 +95,18 @@ protected:
     void SetUp()
     {
         ASSERT_GT(gko::CudaExecutor::get_num_devices(), 0);
+#ifdef GKO_TEST_NONDEFAULT_STREAM
+        cuda = gko::CudaExecutor::create(
+            0, omp, false, gko::default_cuda_alloc_mode, stream.get());
```

Comment:
Maybe I am missing something here. This is still `nullptr` right ? I dont see a `StreamCreate` for association with a non-default stream.

Reply:
no, it is constructed with `cudaStreamCreate` in the `gko::cuda_stream` constructor